docs/source/data-publishing/ogcapi-features.rst

```diff
@@ -240,6 +240,28 @@ This provider has support for the CQL queries as indicated in the Provider table
 .. seealso::
   :ref:`cql` for more details on how to use Common Query Language (CQL) to filter the collection with specific queries.
 
+It is possible to control the order and which properties are exposed/unexposed from a PostGreSQL table using ``properties``, see the example below. 
```

Comment:
Suggest to update text to clarify that `properties` can be applied to any feature (or records) provider that supports.

Reply:
Done - https://github.com/geopython/pygeoapi/pull/1032/commits/e33ca798fe4d9a142e2e6fc4725b24b1fbf0e0f9